modules/keymaps.nix

```diff
@@ -157,21 +22,26 @@ in {
                 either
                 str
                 (
-                  mkMapOptionSubmodule
+                  keymap-helpers.mkMapOptionSubmodule
```

Comment:
Why not put in helpers.keymaps ? like this it avoids to add another kind of special import, and you don't have to add the necessary stuff to make it import automatically in home-assistant/nixos modules & standalone configurations.

Reply:
Very good idea ! Done :)